### DISCUSSION TITLE
How can I get the sample code for each class?

### DISCUSSION THREAD

**Discussion by Ehenglin77** [Cyfrin/foundry-full-course-cu#2050] [General]
How can I get the sample code for each class?

**Comment by EngrPips** [comment]
Hello @Ehenglin77, are you taking the course on Youtube or Cyfrin Updraft?

**Reply by Ehenglin77** [reply]
Youtube

**Reply by EngrPips** [reply]
There should be links in the description of the post. Have you checked that yet?

**Reply by Ehenglin77** [reply]
Sorry, I can't find it. Can you provide me with a link?

**Reply by EngrPips** [reply]
Here https://github.com/Cyfrin/foundry-full-course-cu/, you will see all the codebase for every section of the whole course. Please ask questions if anything is confusing.
